x-pack/plugins/observability_solution/entity_manager/server/lib/entities/ingest_pipeline/generate_history_processors.ts

```diff
@@ -63,6 +63,18 @@ export function generateHistoryProcessors(definition: EntityDefinition) {
         value: definition.id,
       },
     },
+    {
+      set: {
+        field: 'entity.definitionVersion',
+        value: definition.version,
+      },
+    },
+    {
+      set: {
+        field: 'entity.schemaVersion',
+        value: 'v1',
```

Comment:
I'll use the constant for this once https://github.com/elastic/kibana/pull/187507 merges